lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -180,6 +212,55 @@
 '''
 import time
 
+DEPLOYMENT_CONFIGURATION_TYPE_MAP = {
+    'maximum_percent': 'int',
+    'minimum_healthy_percent': 'int'
+}
+
+class TypeMapper:
```

Comment:
@bcoca I've addressed it in my next commit. Let me know if all is good now so we can push this.